### ISSUE TITLE
Add new item on the fly

### ISSUE THREAD

**Issue by Sandvik** [sarperdag/SESpringBoard#4]
Is it possible to add Items when Springboard is loaded? -- Could be nice to add on the fly

**Comment by sarperdag** [comment]
Yeah, actually it'd be better if I add a UITableView like delegate methoding.

**Comment by nanfang** [comment]
we really need it
